### PR TITLE
LPS-168196 Fixed when user info is edited, email is not updated

### PR DESCRIPTION
**Pull request by samuelleolopes**

### CODE REVIEW

**Comment by samuelleolopes** [comment]
ci:test:sf

**Review by kevenleone — APPROVED** [review]
LGTM 🔥

**Comment by kevenleone** [comment]
ci:forward

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Manage/User/UserOutlet.tsx [hunk 1]
```diff
@@ -14,21 +14,31 @@
...
-		userId ? liferayUserAccountsRest.getResource(userId as string) : null
+		userId ? liferayUserAccountsRest.getResource(userId as string) : null,
+		undefined
 	);
 
 	const context = {
-		mutateUser: userId ? mutate : mutateMyUserAccount,
+		mutateUser: userId
+			? userId === Liferay.ThemeDisplay.getUserId()
+				? (response: KeyedMutator<any>) => {
```
Comment (welitonpe) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Manage/User/UserOutlet.tsx:36]: suggestion you can define the type of response.

```suggestion
				? (response: KeyedMutator<UserAccount>) => {
```

**Review by welitonpe — COMMENTED** [review]
Nice :+1:

**Comment by samuelleolopes** [comment]
ci:test:sf

**Comment by welitonpe** [comment]
ci:stop

**Comment by welitonpe** [comment]
ci:forward

**Pull request closed without merging**